plugins/catalog/src/components/AboutCard/AboutContent.tsx

```diff
@@ -34,16 +34,22 @@ export interface AboutContentProps {
    * within EntityLayout, and the entity prop will be removed
    * in future releases.
    *
-   * @deprecated use useEntity hook in the children.
+   * @deprecated use useEntity hook in children.
    */
-  entity: Entity;
+  entity?: Entity;
   children?: React.ReactNode;
 }
 
 // TODO: It seems the whole AboutContent component here is redundant,
 // consider deprecating and removing it.
 /** @public */
 export const AboutContent = (props: AboutContentProps) => {
-  const { children } = props;
+  const { entity, children } = props;
+  if (entity !== undefined) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      'The entity prop is deprecated, use "useEntity" hook in children to access the entity instead',
```

Comment:
I think a more appropriate message would be something like:

Make sure to use this component in an `EntityLayout` or inside an `EntityProvider`